bundle/get_started.md

```diff
@@ -0,0 +1,417 @@
+
+directory = os.environ.get("MONAI_DATA_DIRECTORY")
+root_dir = tempfile.mkdtemp() if directory is None else directory
+compressed_file = os.path.join(root_dir, "Task09_Spleen.tar")
+data_dir = os.path.join(root_dir, "Task09_Spleen")
+if not os.path.exists(data_dir):
+    download_and_extract(resource, compressed_file, root_dir, md5)
+```
+
+## Define train config - Set imports and input / output environments
+
+Now let's start to define the config file for a regular training task. MONAI bundle support both `JSON` and `YAML` format, here we use `JSON` as the example. the whole config for training is available and can be a reference: https://github.com/Project-MONAI/tutorials/blob/main/bundle/spleen_segmentation/configs/train.json.
+
+According to the predefined syntax of MONAI bundle, `$` indicates an expression to evaluate, `@` refers to another object in the config content. For more details about the syntax in bundle config, please check: https://docs.monai.io/en/latest/config_syntax.html.
+
+Please note that a MONAI bundle doesn't require any hard-coded logic in the config, so users can define the config content in any structure.
+
+For the first step, import `os` and `glob` to use in the `python expressions` (start with `$`), then define input / output environments and enable `cudnn.benchmark` for better performance.
+
+The `dataset_dir` in the config is the directory of downloaded dataset. Please check the `root_dir` and update this accordingly when you are writing your config.
+
+Note that the `imports` are only used to execute the `python expressions`, and already imported `monai`, `numpy`, `np`, `torch` internally as these are mininum dependencies of MONAI.
+
+```json
+{
+    "imports": [
+        "$import glob",
+        "$import os",
+        "$import ignite"
+    ],
+    "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "ckpt_path": "/workspace/data/models/model.pt",
+    "dataset_dir": "/workspace/data/Task09_Spleen",
+    "images": "$list(sorted(glob.glob(@dataset_dir + '/imagesTr/*.nii.gz')))",
+    "labels": "$list(sorted(glob.glob(@dataset_dir + '/labelsTr/*.nii.gz')))"
+}
+```
+
+## Define train config - Define network, optimizer, loss function
+
+Define `UNet` of MONAI as the training network, and use the `Adam` optimizer of PyTorch, `DiceCELoss` of MONAI.
+
+An instantiable config component uses `_target_` keyword to define the class / function name or module path, other keys are args for the component.
+
+Note that for all the MONAI classes and functions, we can use its name in `_target_` directly, for any other packages, please provide the `full module path` in `_target_`.
+
+```json
+"network_def": {
+    "_target_": "UNet",
+    "spatial_dims": 3,
+    "in_channels": 1,
+    "out_channels": 2,
+    "channels": [16, 32, 64, 128, 256],
+    "strides": [2, 2, 2, 2],
+    "num_res_units": 2,
+    "norm": "batch"
+}
+```
+
+Move the network to the expected `device`
```

Comment:
I would suggest to explain "device" better here by

> Move the network to the expected `device` which was defined earlier by
> ```"device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')"```

Reply:
updated in the latest commit.

Thanks.